examples/javascript/spi.js

```diff
@@ -24,15 +24,13 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-var m = require('mraa'); //require mraa
+const mraa = require('mraa'); //require mraa
 
-x = new m.Spi(0)
-buf = new Buffer(4)
+let spiDevice = new mraa.Spi(0)
```

Comment:
Lets's add closign semicolons to all lines in this file to align with the proposed style